docs/deployment/README.md

```diff
@@ -24,6 +24,10 @@ Installation steps can be found [here](add-ons/storage/efs)
 ### Using FSx for Lustre with Kubeflow
 Installation steps can be found [here](add-ons/storage/fsx-for-lustre)
 
+
+### CloudWatch Logging and Container Insights
```

Comment:
So there are few pieces here,
apart from collecting logs we also need to collect metrics and publishes to CW. I think you've linked top level doc which is great, can we also call it here what all they can do. Currently we are talking about logging only.